templates/hp_comware_display_counters_bound_interface.template

```diff
@@ -0,0 +1,17 @@
+Value INTERFACE (\S+)
+Value PACKETS_TOTAL (\d+)
+Value PACKETS_BROADCAST (\d+)
+Value PACKETS_MULTICAST (\d+)
+Value PACKETS_ERROR (\d+)
+
+Start
+  ^\s*$$
+  ^Interface\s+Total\s+\(pkts\)\s+Broadcast\s+\(pkts\)\s+Multicast\s+\(pkts\)\s+Err\s+\(pkts\) -> InterfaceTable
+
```

Comment:
```suggestion
  ^. -> Error
```
Should error in case a different format of the table is produced by the command output